index.html

```diff
@@ -1,12 +1,161 @@
+        <li class="menu_items"><a href="#">Sponsors</a></li>
+      </ul>
+    </div>
+  </header>
+
+  <section id="content">
+    <div class="home-top">
+      <div class="overlay">
+        <div class="home-text">
+          <h3>"It's about time to connect with your roots?"</h3><br />
+          <h1>BACK TO ROOTS<br /> GLOBAL CONCERT <br /> 2012</h1>
+        </div>
+        <div class="home-p">
+          <p>A joyful celebration believing in the value of connecting with one's roots through musical entertainment.
+            Become the part of this year's concert by buying your ticket online.</p>
+        </div>
+        <div class="home-p2">
+          <span>2012.10.15(FRI) ~ 17(SUN)</span>
+          <p>@Madison Square Garden, New York City, USA</p>
+        </div>
+      </div>
+    </div>
+
+    <div class="home2 container-fluid" id="program">
+      <p>Main Program <img class="line" src="assets/images/line_icon5.png" alt="line"> </p>
+      <div class="card-box container-fluid">
+        <div class="box box1 container-fluid">
+          <div class="icon icon1 no-hover">
+            <img class="no-hover" src="assets/images/music2_icon.svg" alt="microphone">
+          </div>
+          <div class="head6 no-hover">
+            <h6 class="no-hover">Music</h6>
+          </div>
+          <div class="box-p no-hover">
+            <p class="no-hover">Listen to prominent musicians from various countries sharing the message of love through
+              their music.</p>
+          </div>
+        </div>
+        <div class="box box2 container-fluid">
+          <div class="icon icon2 no-hover">
+            <img class="no-hover" src="assets/images/exhibition_icon.svg" alt="exhibition icon">
+          </div>
+          <div class="head6 no-hover">
+            <h6 class="no-hover">Exhibition</h6>
+          </div>
+          <div class="box-p no-hover">
+            <p class="no-hover">There will be a lot of items on exhibition ranging from fashoin to art and promotions.
+            </p>
+          </div>
+        </div>
+        <div class="box box3 container-fluid">
+          <div class="icon icon3 no-hover">
+            <img class="no-hover" src="assets/images/music_icon.svg" alt="music box icon">
+          </div>
+          <div class="head6 no-hover">
+            <h6 class="no-hover">Entertainment</h6>
+          </div>
+          <div class="box-p no-hover">
+            <p class="no-hover">Entertainment is core factor of the Back To Roots Global Concert, and be ready to be
+              blown away.</p>
+
+          </div>
+        </div>
+        <div class="box box4 container-fluid">
+          <div class="icon icon4 no-hover"><img class="no-hover" src="assets/images/donation_icon.svg"
+              alt="dollar coin icon"></div>
+          <div class="head6 no-hover">
+            <h6 class="no-hover">Pledges <br /> & <br /> Donations</h6>
+          </div>
+          <div class="box-p no-hover">
```

Comment:
[OPTIONAL]  I am only marking this as optional because the previous reviewer missed it. The background for this section does not match Cindy's design